pkg/ddevapp/templates.go

```diff
@@ -155,11 +165,13 @@ networks:
 volumes:
   mariadb-database:
     name: "${DDEV_SITENAME}-mariadb"
+  {{ if .OmitSSHAgent }}
```

Comment:
Needs a `not` here

Reply:
Wow, and here I thought I tested that successfully. Thanks.